### ISSUE TITLE
Add a list of different things new volunteers can work with.

### ISSUE THREAD

**Issue by Lemoert** [Trustroots/community#45]

**Comment by loberto** [comment]
Do we want to add a list on this issue? with check list?
Or do we want different task on a separate project?

**Comment by Lemoert** [comment]
If we set up a project that contains all the small tasks new volunteers can have a look at it might be superfluous to have a list for this..

**Comment by simison** [comment]
Much like https://team.trustroots.org/Volunteering.html ?

**Comment by Lemoert** [comment]
Yup, similar to that but less generic. A collection of small actual issues that a new volunteer can have a look at and help out with, if that makes sense.
